tensorflow/stream_executor/device_description.h

```diff
@@ -78,6 +78,10 @@ class DeviceDescription {
   // legitimate kernel launch request.
   const BlockDim &block_dim_limit() const { return block_dim_limit_; }
 
+  // Returns the limit on the number of simultaneously resident blocks
```

Comment:
Maybe s/limit on the number/max/?

Reply:
Sounds like a good change to me (This was the original comment before the previous PR removed the method)